psd-web/db/migrate/20200507145004_drop_teams_users.rb

```diff
@@ -0,0 +1,37 @@
+class DropTeamsUsers < ActiveRecord::Migration[5.2]
+  def up
+    # Check that all users have one team
+    safety_assured do
+      query = execute "SELECT count(*) FROM users WHERE (SELECT count(*) AS teams_count FROM teams_users WHERE teams_users.user_id=users.id) != 1"
+      raise "Users with zero or more than one team exist - clean the data first" if query.first["count"] != 0
+    end
+
+    add_column :users, :team_id, :uuid, index: true
```

Comment:
Add a foreign key here too?

```suggestion
    add_column :users, :team_id, :uuid, index: true
    add_foreign_key :users, :teams
```